### ISSUE TITLE
Xcode 16b5: Undefined symbols for architecture arm64

### ISSUE THREAD

**Issue by PSKuznetsov** [myTrackerSDK/mytracker-ios#7]
Hi there!
I encountered an issue where the library does not compile in Xcode 16 beta 5 with following error:
```
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_ADClient", referenced from:
       in MyTrackerSDK[arm64][62](MRIadHandler.o)
ld: symbol(s) not found for architecture arm64
```

**Comment by PSKuznetsov** [comment]
As can I see ADClient was marked as OBSOLETED in iOS 18
```
#define ADCLIENT_DEPRECATED_IOS_71_145_OBSOLETED_180 __attribute__((availability(ios, introduced=7.1, deprecated=14.5, obsoleted=18.0, \
    message="This has been replaced by functionality in AdServices.framework's AAAttribution class.")))
```

```
ADCLIENT_DEPRECATED_MACOS_1014_113_OBSOLETED_180 ADCLIENT_DEPRECATED_IOS_71_145_OBSOLETED_180 ADCLIENT_DEPRECATED_TVOS_71_145_OBSOLETED_180
@interface ADClient : NSObject
```

**Comment by aserrr** [comment]
Hi,
Thank you for noting this issue.
We'll update SDK in near future and replace this deprecated api.

**Comment by PSKuznetsov** [comment]
Resolved in 3.2.0 
Thanks!
